### PR TITLE
HAAR-895: Add sqlserver config and flyway to add hpa stage data

### PR DESCRIPTION
**Pull request by SimonMitchellMOJ**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/historicalprisonerindexer/model/Addresses.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import java.io.Serializable
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity
+@Table(name = "ADDRESSES")
+data class Addresses(
```
Comment (karenmillermoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/historicalprisonerindexer/model/Addresses.kt:12]: I think the class should be called Address - the table should be too - but I'm assuming we're not changing that

Comment (SimonMitchellMOJ) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/historicalprisonerindexer/model/Addresses.kt:12]: I am going to port everthing over `as is` - this was just to get the sqlserver setup in

**Review by karenmillermoj — DISMISSED** [review]
lgtm

**Review by karenmillermoj — APPROVED** [review]
lgtm

**Pull request merged**